### PR TITLE
Change the default of --container-state to `all`

### PR DESCRIPTION
**Pull request by tksm**
This PR changes the default value of --container-state to [`all`](https://github.com/stern/stern/pull/222). It will be a breaking change.

Currently, the default value is `running`, so it does not show logs of completed pods (`terminated`) and CrashLoopBackoff pods (`waiting`). It is a bit [confusing](https://github.com/stern/stern/issues/36#issuecomment-1108906137) because `kubectl logs` shows logs regardless of their container statuses.

I don't know the exact reason why the default is `running`, but I guess it is because stern did not handle terminated containers well. By #221, stern can now handle terminated containers, so I would like to consider changing the default to `all`.

### CODE REVIEW

**Review by rkmathi — COMMENTED** [review]
This is indeed a breaking change, but I think it useful.

@superbrothers @floryut 
What do you think?

**Review by superbrothers — APPROVED** [review]
I am concerned that if there are many Pods created from Jobs, their log output will make it difficult to see the desired log. However, this should not be a problem since we can simply change pod-query or container-state.

**Comment by tksm** [comment]
@floryut Do you have any concerns about this change? It will be a breaking change, so I would like to get a consensus from all maintainers.

**Comment by tksm** [comment]
@floryut I'll merge this PR once but feel free to comment if you have any concerns about this change.

**Comment by floryut** [comment]
Sorry I was away for 10 days 🎿 

I'm fine with this, this is a breaking change, but I'll think it's better to follow the same logic as kubectl as long as the logic is not buggy/faulty

**Pull request merged**
